src/foundation.cs

```diff
@@ -9832,9 +9832,22 @@ partial interface NSBundle {
 		[Export ("bundleWithIdentifier:")][Static]
 		NSBundle FromIdentifier (string str);
 
+#if !XAMCORE_5_0
+		[Internal]
 		[Export ("allBundles")][Static]
+		NSArray _InternalAllBundles { get; }
+
+		[Obsolete ("Use the 'allBundles' property instead.")]
```

Comment:
We use the C# name in obsolete messages:

```suggestion
		[Obsolete ("Use the 'AllBundles' property instead.")]
```